components/nimbus/ios/Nimbus/NimbusCreate.swift

```diff
@@ -77,34 +77,31 @@ public extension Nimbus {
     /// - Throws `NimbusError` if anything goes wrong with the Rust FFI or in the `NimbusClient` constructor.
     ///
     static func create(
-        _ server: NimbusServerSettings?,
         appSettings: NimbusAppSettings,
         coenrollingFeatureIds: [String] = [],
         dbPath: String,
         resourceBundles: [Bundle] = [Bundle.main],
         enabled: Bool = true,
         userDefaults: UserDefaults? = nil,
         errorReporter: @escaping NimbusErrorReporter = defaultErrorReporter,
-        recordedContext: RecordedContext? = nil
+        recordedContext: RecordedContext? = nil,
+        remoteSettingsService: RemoteSettingsService? = nil,
+        collectionName: String? = nil
```

Comment:
This will require an associated change here https://github.com/mozilla/application-services/blob/b56ecd38f1e76c1fc36f2fe94ac9a1035b68bf87/components/nimbus/ios/Nimbus/NimbusBuilder.swift#L262-L271